packages/vuetify/src/components/VAutocomplete/VAutocomplete.tsx

```diff
@@ -386,7 +386,7 @@ export const VAutocomplete = genericComponent<new <
       } else {
         if (!props.multiple && search.value == null) model.value = []
         menu.value = false
-        if (!model.value.some(({ title }) => title === search.value)) search.value = ''
+        if (props.multiple || hasSelectionSlot.value) search.value = ''
```

Comment:
Looks like this change broke the search functionality. When I type anything it gets cleared immediately.

Minimal reproduction:

```
<v-autocomplete
    v-model="customer"
    v-model:search="searchCustomer"
    :items="customers"
    variant="outlined"
>
</v-autocomplete>
```

Reply:
I have same issue and had to downgrade back to previous version

Reply:
#21392 - already fixed. Will be released in v3.8.5. You can downgrade to v3.8.3 or use `patch-package` to fix current version on the fly.

Reply:
Thanks @J-Sek